vms/platformvm/block/executor/verifier.go

```diff
@@ -65,20 +65,16 @@ func (v *verifier) BanffProposalBlock(b *block.BanffProposalBlock) error {
 		return err
 	}
 
-	// Apply the changes, if any, from advancing the chain time.
-	nextChainTime := b.Timestamp()
-	changes, err := executor.AdvanceTimeTo(
+	// Advance the time to [b.Timestamp()].
+	_, err = executor.AdvanceTimeTo(
```

Comment:
same nit as above about reducing scope
```suggestion
	if _, err := executor.AdvanceTimeTo(
```